queries/gnuplot/highlights.scm

```diff
@@ -0,0 +1,583 @@
+; highlights.scm
+; FIX: "tc" "fc" "lc" highlight
+
+(comment) @comment
+
+(unary_expression
+  _? @operator
+  (_) @variable
+  _? @operator)
+
+(binary_expression
+  (_) @variable
+  _ @operator
+  (_) @variable)
+
+(ternary_expression
+  condition: (_) @variable
+  "?" @conditional.ternary
+  true: (_) @variable
+  ":" @conditional.ternary
+  false: (_) @variable)
+
+(func_def (function) "=" @operator)
+(var_def var: (identifier) "=" @operator)
+(array_def "array"
+           [
+            ((array) ("=" @operator)?)
+            ((identifier) "[" (_) "]" "=" @operator)
+            ])
```

Comment:
```suggestion
"=" @operator
```

You usually don't have to query entire branches, just the relevant nodes. Same for all the queries below.